### ISSUE TITLE
High validation loss for depth prediction

### ISSUE THREAD

**Issue by namsan96** [mees/hulc2#3]
Hi, thanks for sharing your codebase for great work!

I'm trying to reproduce HULC++'s evaluation results in CALVIN.
I found that the validation loss of depth prediction increases very highly compared to the training's one in affordance model training, as shown in the figure below.
It'd be very helpful if you can confirm whether this is an expected behavior, or if I'm missing something while following your instructions.

I actually encountered very suspicious error during the final stage of processing validation data.
The error occurred in [this line](https://github.com/mees/hulc2/blob/bc3f7e087ba62a2bad6928314bc394402cd2c9b4/hulc2/affordance/dataset_creation/data_labeler_lang.py#L109C9-L109C24) as [add_norm_values](https://github.com/mees/hulc2/blob/bc3f7e087ba62a2bad6928314bc394402cd2c9b4/hulc2/affordance/dataset_creation/find_norm_values.py#L23) tries to indexing `data['training']` which appears to be reasonably missing while processing validation data.
Would this error have affected data sanity?

Thanks!

![Screenshot from 2023-08-07 13-11-21](https://github.com/mees/hulc2/assets/17007461/db3e793b-ca19-44dc-9b74-be850325c3b9)

**Comment by HuiHuiSun** [comment]
> Hi, thanks for sharing your codebase for great work!
> 
> I'm trying to reproduce HULC++'s evaluation results in CALVIN. I found that the validation loss of depth prediction increases very highly compared to the training's one in affordance model training, as shown in the figure below. It'd be very helpful if you can confirm whether this is an expected behavior, or if I'm missing something while following your instructions.
> 
> I actually encountered very suspicious error during the final stage of processing validation data. The error occurred in [this line](https://github.com/mees/hulc2/blob/bc3f7e087ba62a2bad6928314bc394402cd2c9b4/hulc2/affordance/dataset_creation/data_labeler_lang.py#L109C9-L109C24) as [add_norm_values](https://github.com/mees/hulc2/blob/bc3f7e087ba62a2bad6928314bc394402cd2c9b4/hulc2/affordance/dataset_creation/find_norm_values.py#L23) tries to indexing `data['training']` which appears to be reasonably missing while processing validation data. Would this error have affected data sanity?
> 
> Thanks!
> 
> ![Screenshot from 2023-08-07 13-11-21](https://private-user-images.githubusercontent.com/17007461/258697669-db3e793b-ca19-44dc-9b74-be850325c3b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxODgzMDksIm5iZiI6MTcxMzE4ODAwOSwicGF0aCI6Ii8xNzAwNzQ2MS8yNTg2OTc2NjktZGIzZTc5M2ItY2ExOS00NGRjLTliNzQtYmU4NTAzMjVjM2I5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE1VDEzMzMyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmYTY3OWYyYzEwYzgxMmMwZTM2YWYyOTg0NTUzMzlkMzhjNzZlMzA2NDNmNzRmNTg4OWJjMTNkZTE5ZDdlZWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qGhUJ6GV6RIIgOKaCrACDtjbCc0D0ityU1d1X2REJqY)

Hi, have you reproduced the results in the paper?
